x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/bulk_delete.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { HttpSetup } from '@kbn/core/public';
+import { KueryNode } from '@kbn/es-query';
+import { INTERNAL_BASE_ALERTING_API_PATH } from '../../constants';
+import { BulkDeleteResponse } from '../../../types';
+
+export const bulkDeleteRules = async ({
+  filter,
+  ids,
+  http,
+}: {
+  filter?: KueryNode | null;
+  ids?: string[];
```

Comment:
as the father component sets idsToDelete as string[], is the optional really needed?

Reply:
f51e42a856b49e6f644d372bfbc547a1a1e2914f